cookbook/ui_customization/change_pim_locale.rst

```diff
@@ -0,0 +1,28 @@
+How to change the PIM locale
+============================
+
+In this cookbook, you will learn how to manage Akeneo PIM locales.
+
+How to allow more locales
+-------------------------
+
+The global Akeneo PIM locale and user interface locales are configurable. By default, the Akeneo PIM is configured to
+only allow locales translated to more than 80%.
+
+If you want to allow more locales to be accessible, you can configure this limit in
+``src/Pim/Bundle/LocalizationBundle/Resources/config/providers.yml`` file:
+
+.. code-block:: yaml
+
+    parameters:
+        pim_localization.provider.ui_locale.class:          Pim\Bundle\LocalizationBundle\Provider\UiLocaleProvider
+        pim_localization.provider.ui_locale.min_percentage: 0.8
+
+For example, if you want to allow locales translated to more than 70%, replace the minimal percentage ``0.8`` by
+``0.7``.
+
+How to improve translations
+---------------------------
+
+If your locale is not allowed or if you need to add missing translations, you can fill the related ``messages.yml`` or,
+better, contribute and :doc:`/contributing/translate`.
```

Comment:
Seen a few times in other templates (e.g. https://github.com/akeneo/pim-docs/blob/master/contributing/create_connector.rst ... It generates a simple link)